### PR TITLE
minor: change page hints for number and text components

### PR DESCRIPTION
**Pull request by JamesBoadi**
# Description

Page hint misleads users into using `govuk-input-width` to change the width of number and field components when is not available as a css class. Changing page hints to correctly reflect the most recent documentation [styles spacing](https://design-system.service.gov.uk/styles/spacing/#width-override-classes) [#759 ](https://github.com/XGovFormBuilder/digital-form-builder/issues/759)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My changes do not introduce any new linting errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and versioning
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have rebased onto main and there are no code conflicts
- [ ] I have checked deployments are working in all environments
- [ ] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

**Review by RichardBray — APPROVED** [review]

**Review by RichardBray — APPROVED** [review]
LGTM

**Comment by jenbutongit** [comment]
LGTM also 👍 (merging after actions have run, if I forget, please merge)

**Pull request merged**
